plugins/tracers/gstctf.c

```diff
@@ -452,10 +457,17 @@ tcp_parser_handler (gchar * line)
     ++line_end;
     port_name = line_end;
 
-    /* TODO: verify if is a numeric string */
     ctf_descriptor->port_number = g_ascii_strtoull (port_name,
         &port_name_end, 10);
 
+    /* Verify if the convertion of the string works */
+    if ('\0' != *port_name_end && '-' == port_name[0]) {
+      ctf_descriptor->port_number = SOCKET_PORT;
+      GST_ERROR
+          ("Set it an invalid value of port number, using the default value: %d",
```

Comment:
Let's print the port name as well: `GST_ERROR ("Invalid port number \"%s\", using the default value: %d", port_name, ctf_descriptor->port_number);`